setup.py

```diff
@@ -34,18 +34,23 @@ def get_extras_require():
         "tools",
         "all-contrib-expectations",
     )
+
     requirements_dir = "reqs"
-    rx_fname_part = re.compile(rf"{requirements_dir}/requirements-dev-(.*).txt")
-    for fname in glob(f"{requirements_dir}/*.txt"):
-        match = rx_fname_part.match(fname)
+    rx_name_part = re.compile(rf"requirements-dev-(.*).txt")
+
+    # Use Path() from pathlib so we can make this section of the code OS agnostic.
+    # Loop through each requirement file and verify they are named
+    # correctly and are in the right location.
+    for path_and_file_name in Path().glob(f"{requirements_dir}/*.txt"):
```

Comment:
```suggestion
    for file_path in Path().glob(f"{requirements_dir}/*.txt"):
```
Minor point.
`file_name` makes me think it's a string, but it's a `Path` object.